File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

// Declaring Functions
int input(void);
void print_pattern(int n);

// Main Function
int main(void)
{
    int width = input();
    print_pattern(width);
}

// Taking input from user
int input(void)
{
    int n;
    do
    {
        n = get_int("Enter Width: ");
    }
    while (n <= 0 || n > 8);
    return n;
}

// Printing Pattern
void print_pattern(int n)
{
    for (int i = 0; i < n; i++)
    {
        // Printing first part of pattern
        for (int j = i; j < (n - 1); j++)
        {
            printf(" ");
        }
        for (int j = n; j >= (n - i); j--)
        {
            printf("#");
        }
        printf("  ");

        // Printing second part of pattern
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Getting input, height, from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Printing desired pattern
cpy1 = height
cpy2 = height
for i in range(height):

    # Printing first part of pattern
    j = 1
    while j < cpy1:
        print(" ", end="")
        j += 1
    while j >= cpy1 and j <= height:
        print("#", end="")
        j += 1
    cpy1 -= 1
    print("  ", end="")

    # Printing second part of pattern
    k = height
    while k <= cpy2:
        print("#", end="")
        k += 1
    cpy2 += 1
    print("")